edge/pkg/metamanager/metaserver/server.go

```diff
@@ -63,6 +67,16 @@ func (ls *MetaServer) Start(stopChan <-chan struct{}) {
 		}
 	}()
 
+	client.InitKubeEdgeClient()
+
+	util.CRDMapper = meta.NewDefaultRESTMapper([]schema.GroupVersion{{Group: "apiextensions.k8s.io", Version: "v1beta1"}})
+	go wait.Until(func() {
+		err := util.UpdateCrdMap()
+		if err != nil {
+			klog.Warningf("CRD Resource-Kind Map update failed: %v", err)
+		}
+	}, time.Second*30, stopChan)
```

Comment:
I don’t think that regular update is a better solution. 
Do you consider event triggering?
When receiving the crd type resource creation and when the service starts, synchronize it.

Reply:
Yes, the loop detection is only a temporary solution to deal with the problems previously reported.
Event triggering is indeed a more reasonable solution. I thought about it....The solution can be like this: 
Since EdgeController does not support CRD, and dynamicController needs to receive watch requests to create an informer for X-resource, it may need to be started on MetaServer. Send a watch request for `apiextensions.k8s.io/v1beta1/CustomResourceDefinition`, so that every time the crd resource is created(whether through `kubectl apply` or code creation), the dynamic controller will be notified. I call `UpdateCrdMap()` to update when the dynamic controller receives the event. How do you think?

Reply:
Well, and when the `edgecore` is started for the first time, it needs to be synchronized once from the storage.

Reply:
Yes, when `MetaServer` starts, it sends a watch request to synchronize crd resources... If there is no problem, I can try to implement it?

Reply:
I mean, because `watch` is incremental monitoring, when `edgecore` restarts, does it need to be fully synchronized once?

Reply:
No, because I will list the full crd in the response logic issued by the synchronously triggered event.
And, I will place the logic that triggers the update of the map in the process of MetaManager receiving msg, because it is convenient for testing:)

Reply:
@zc2638   already updated, plz review

Reply:
@zc2638   been waiting  (*❦ω❦)